### DISCUSSION TITLE
test that instalation is working without inverter.

### DISCUSSION THREAD

**Discussion by estqwerty** [jblance/mpp-solar#187] [Ideas]
Hi there , i think that I installed it right , but my inverter is still not here. Is there a way to generate fake/test input from mpp-solar just to check that everything is working. Tanks in advance.

**Comment by jblance** [comment]
not easily
there is a test function using ```-p test``` that will simulate a response and you could try to simulate a serial port using linux tools but thats about it

**Reply by estqwerty** [reply]
Thank You will give it a try.
